Mathlib/Geometry/Manifold/VectorBundle/SmoothSection.lean

```diff
@@ -227,6 +227,63 @@ lemma ContMDiffOn.smul_section_of_tsupport {s : Π (x : M), V x} {ψ : M → 
     simp [image_eq_zero_of_notMem_tsupport hy, zeroSection]
   · exact Set.compl_subset_iff_union.mp <| Set.compl_subset_compl.mpr ht'
 
+/-- The sum of a locally finite collection of sections is `C^k` iff each section is.
+Version at a point within a set. -/
+lemma ContMDiffWithinAt.sum_section_of_locallyFinite {ι : Type*} {t : ι → (x : M) → V x}
```

Comment:
maybe have iota and t as section variables, as they are used in several statements.